src/js/content/index.js

```diff
@@ -90,7 +90,11 @@ function onSubmitForm(
     const isIgnored = ignoredSites.some(is => is === domain);
     console.log('[subscriptionscore]: ranked', `${rank} - ${domain}`);
 
-    if (!isIgnored && ranks.indexOf(rank) <= ranks.indexOf(blockedRank)) {
+    const isBelowAlertThreshold =
+      !!rank && ranks.indexOf(rank) <= ranks.indexOf(blockedRank);
```

Comment:
remove `!!rank` redundant code since `ranks.indexOf(undefined)` with always be `-1` and outcome will be the same.